data/json/effects.json

```diff
@@ -1309,6 +1310,10 @@
             "Trashed",
             "Wasted"
         ],
+        "desc": [ "You have drunk some alcohol.  You feel warm inside.",
```

Comment:
The grammar is a bit stilted; recommend `"You drank some alcohol.` for this and the ones below, followed by the text.